scalafix-rules/src/main/scala-2/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -69,184 +74,64 @@ final class ExplicitResultTypes(
     if (
       config.scalacClasspath.nonEmpty && inputBinaryScalaVersion != runtimeBinaryScalaVersion
     ) {
-      Configured.error(
-        s"The ExplicitResultTypes rule needs to run with the same Scala binary version as the one used to compile target sources ($inputBinaryScalaVersion). " +
-          s"To fix this problem, either remove ExplicitResultTypes from .scalafix.conf or make sure Scalafix is loaded with $inputBinaryScalaVersion."
-      )
```

Comment:
If not for scala 3. we still need that to prevent trying to load Scala 2.12 against 2.13 or the other way around, right?